app/src/main/java/org/mozilla/focus/activity/RightsActivity.java

```diff
@@ -1,6 +1,34 @@
 package org.mozilla.focus.activity;
 
-import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.webkit.WebView;
 
-public class RightsActivity extends Activity {
+import org.mozilla.focus.R;
+
+public class RightsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_rights);
+        WebView mWebView = null;
+        mWebView = (WebView) findViewById(R.id.webview);
```

Comment:
It would be nice if we could use WebViewProvider here - so that we use the same rendering engine depending on build type. But that's something we could do in a follow-up issue.